lib/ramble/ramble/package_manager.py

```diff
@@ -187,7 +197,7 @@ def build_used_variables(self, workspace):
 
         software_environments = workspace.software_environments
         software_environments.render_environment(
-            app_context, self.app_inst.expander, require=False
+            app_context, self.app_inst.expander, self.app_inst.package_manager, require=False
```

Comment:
I think `self.app_inst.package_manager` should just be `self` (they should point to the same thing).

I don't think it's actually wrong to use this, but it's a bit longer than it needs to be (and also makes it feel like `self` shouldn't be a package manager).

Reply:
Good catch, updated.